FreeRTOS/FreeRTOS
# Add a hardware definition project for the MicroZed board

Description
-----------
Add a hardware definition project for the MicroZed board to the existng Zynq ZC702 project.
Add a text file that describes how to switch the Zynq project form the ZC702 hardware to the MicroZed hardware.

Test Steps
-----------
No functional changes.

Related Issue
-----------
None


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
